REQUIRE

```diff
@@ -1,5 +1,5 @@
-julia 0.4
-Compat 0.7.16
+julia 0.5
+Compat 0.16.0
```

Comment:
what are you using that was first in 0.16? looks like 0.9.5 might be good enough for `take!`